entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+exec gunicorn \
+  -u nobody \
+  -g nobody \
+  --access-logfile '-' \
+  --error-logfile '-' \
+  --log-file "-" \
+  --access-logformat '%({x-forwarded-for}i)s %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"' \
+  --forwarded-allow-ips="140.211.9.104,2605:bc80:3010:104::8cd3:968" \
```

Comment:
I may send a follow-up PR that breaks these values out into a variable so they can be described in a comment for future me who forgets what these IPs are.